README.md

```diff
@@ -42,6 +44,27 @@ npm start
 This will start up a server running on port 3000 with a backend storing all data in memory.
 More configs with different backends can be found in the config folder.
 
+## Running inside a Docker container
+
+Alternatively, you can run the community server inside a Docker container. Start by building a Docker image based on the `Dockerfile`.
+
+```
+docker build -t solid/community-server:0.1.1 .
```

Comment:
I would exclude the version number here (and below), as we'll probably forget to update this version in the future.

Reply:
Also, we may want to consider auto-building on Docker hub.